### DISCUSSION TITLE
Gujarati font not rendering properly in MPDF 8.1.5

### DISCUSSION THREAD

**Discussion by nishantmonapara** [mpdf/mpdf#1871] [General]
I'm converting HTML/CSS to PDF, There is an issue with font rendering,
I'm adding આદિનાથ ભગવાન and getting the wrong output  
![image](https://github.com/mpdf/mpdf/assets/36466968/740db40d-0f3e-4382-a390-380e0bf565c9)

**Comment by dineshkapadiya** [comment]
same problem can't find any solution in 2025

**Comment by thehemantparmar** [comment]
use below link its working first created page in html then url and its convert gujrati text to pdf perfectly.
*** you need chrome browser for this***
 
https://github.com/DaanDeSmedt/PHPHeadlessChrome
